backend/demo/src/main/java/com/example/demo/controller/TransportationController.java

```diff
@@ -0,0 +1,54 @@
+package com.example.demo.controller;
+
+import com.example.demo.model.Transportation;
+import com.example.demo.exception.ResourceNotFoundException;
+import com.example.demo.repository.TransportationRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+import java.util.Optional;
+
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/transportation")
+
+
+public class TransportationController {
+
+   
+    @Autowired
+    private TransportationRepository transportationRepository;
```

Comment:
create a service layer that the controller will talk to, instead of directly accessing the repository